operatorconfig/moduleconfig/replication/v1.7.0/container.yaml

```diff
@@ -0,0 +1,24 @@
+name: dell-csi-replicator
```

Comment:
Could you remove 1.4.0 version replication files as we support n-2 versions in CSM Operator?

operatorconfig/moduleconfig/common/version-values.yaml also needs update.

Reply:
Done